kickoff/jsonexport.py

```diff
@@ -167,29 +167,43 @@ def firefoxVersionsJson():
     return returnJSONVersionFile('firefox_versions.json', versions)
 
 
+def _generateFileSizeMetaData(lastVersion):
+    version = dict()
+    version[lastVersion] = dict()
+    oses = ["Windows", "OS X", "Linux"]
+    for OS in oses:
+        # We insert filesize to backward compatibility
+        version[lastVersion][OS] = {"filesize": 0}
+    return version
+
+
 def generateLocalizedBuilds(buildsVersionLocales, l10nchangesets, lastVersion):
     # The l10n format for desktop is 'locale changeset'
     # parse it
     locales = parsePlainL10nChangesets(l10nchangesets)
+
     for localeCode in locales:
+        version = _generateFileSizeMetaData(lastVersion)
         if localeCode not in buildsVersionLocales.keys():
-            buildsVersionLocales[localeCode] = [lastVersion]
+            buildsVersionLocales[localeCode] = version
```

Comment:
nope, it was creating a [ ] struct in the json causing the file to be different from what we have in p-d